### ISSUE TITLE
Do not show placeholder for a screenshot if there is none

### ISSUE THREAD

**Issue by probonopd** [probonopd/gs-plugin-appimage#4]
Is there a way to not waste that space, like setting `screenshot=none` somethere? cc @hughsie 

![screenshot](https://user-images.githubusercontent.com/2480569/40775609-0664f12e-64b8-11e8-95df-4aadb50e8211.png)

**Comment by hughsie** [comment]
I think the idea is to actually use a screenshot for all desktop apps :) I don't know if xchat provides an appdata file upstream (I thought X-Chat was dead upstream and riddled with security problems) but if it does it should be easy to show the right thing.

**Comment by probonopd** [comment]
> I think the idea is to actually use a screenshot for all desktop apps :) I

I agree with that idea, but reality is that not all applications even have AppStream metainfo files, let alone screenshots...

**Comment by hughsie** [comment]
In GNOME Software we made the conscious choice of only showing components with metadata. One extra file is a pretty low bar really.

**Comment by probonopd** [comment]
Okay, so what we might do is load our automatically-captured screenshots from the automated tests on [AppImageHub](https://appimage.github.io/) if there is no AppStream metadata file or screenshot provided by upstream. In many cases they are not beautiful, but I guess better than nothing.
